src/coreComponents/dataRepository/CMakeLists.txt

```
@@ -44,14 +46,14 @@ if( ENABLE_PYGEOSX )
   set( dependencyList ${dependencyList} pylvarray )
 endif()
 
-blt_add_library( NAME                  dataRepository
-                 SOURCES               ${dataRepository_sources}
-                 HEADERS               ${dataRepository_headers}
-                 DEPENDS_ON            ${dependencyList}
-                 OBJECT                ${GEOSX_BUILD_OBJ_LIBS}
+blt_add_library( NAME       dataRepository
+                 SOURCES    ${dataRepository_sources}
+                 HEADERS    ${dataRepository_headers}
+                 DEPENDS_ON ${dependencyList}
+                 OBJECT     {GEOSX_BUILD_OBJ_LIBS}
```

This typo is the likely reason for CI build failures:
```suggestion
                 OBJECT     ${GEOSX_BUILD_OBJ_LIBS}
```